docs/user/dashboard/vega-reference.asciidoc

```diff
@@ -234,8 +234,12 @@ configuration:
       "longitude": -74,   // default 0
       "zoom": 7,          // default 2
 
-      // defaults to "default". Use false to disable base layer.
-      "mapStyle": false,
+      // Use "mapStyle" to configure a base map layer for your vega map. 
+      // Can be set to one of the following:
+      // - set to false to disable base layer;
+      // - set to "default" to use Kibana's default base map layer;
+      // - set "mapStyle" to any Elastic Maps Service (EMS) tile layer id.  
```

Comment:
How about listing out valid tile layer ids here?

Reply:
we cannot guarantee that this doc will be updated on updating available EMS layers =)

Reply:
I think that is OK, and we can remove once https://github.com/elastic/ems-landing-page/issues/304 is fixed and there is an easy way for users to get the ids from EMS. I will add a comment in https://github.com/elastic/ems-landing-page/issues/304 to update these docs to link to available EMS baselayer ids once they are available.

Reply:
I'm still not a big fan of hard coding that id's. I see that currently for some reasons `maps_legacy/ServiceSettings` returns only `road_map` and in case if user set the value of `map.tilemap.url` I also see `TMS in config/kibana.yml` in that array:

![image](https://user-images.githubusercontent.com/20072247/105153001-f7394380-5b18-11eb-9110-931b5df079dd.png)

I prefer to keep this message as it for now. But in parallel we are working on migration `leaflet -> mapbox` where we use @elastic/ems-client directly. So very soon "road_map_desaturated" and "dark_map" will be also available. 

The goal of that PR is to remove wrong `if` statement from parser.